usagov/vote-gov-drupal
# VOTE-949: Style refinements to CTA text + image

## Jira ticket

VOTE-949

## Description

Adds breakpoints matching this figma reference:
https://www.figma.com/file/E70piAGXmi4UzrHs0A90cm/Vote.gov-Redesign-2024-(Master)?node-id=4697%3A2991&mode=dev

## Deployment and testing

### Pre-deploy

1. Insert any required steps to take before merging and running post-deploy.

### Post-deploy

1. Insert any required steps to take before beginning test.

### QA/Test

1. lando retune
2. npm run build
3. make sure there is a CTA block with content in it
4. check styling against figma file and test with different content lengths

## Checklist for the Developer




## Checklist for the Peer Reviewers
